### ISSUE TITLE
[patch] Incorrect uses of the (sys:simple-int32-vector 256) in quicklisp/deflate.lisp

### ISSUE THREAD

**Issue by m-lw** [quicklisp/quicklisp-client#168]
The declarations using `(sys:simple-int32-vector 256)` at https://github.com/quicklisp/quicklisp-client/blob/cadecc5e4b1923e707ce1bc1bbbafa0e334ae742/quicklisp/deflate.lisp#L128 and https://github.com/quicklisp/quicklisp-client/blob/cadecc5e4b1923e707ce1bc1bbbafa0e334ae742/quicklisp/deflate.lisp#L177 are incorrect and may lead to warnings. The type takes no parameters so this should be `sys:simple-int32-vector` instead.

Patch in https://gist.github.com/m-lw/60af4aeb31067c5558eeee9e13b3eb20.

**Comment by quicklisp** [comment]
Thanks for the patch. Do you happen to have a link to the documentation for this type?

**Comment by m-lw** [comment]
Yes, it is here: http://www.lispworks.com/documentation/lw71/LW/html/lw-1451.htm

**Comment by priyadarshan** [comment]
I am experiencing same warning using LispWorks 7.1.1 on Windows 10:

```
Ignoring type declaration with illegal type (SIMPLE-INT32-VECTOR 256)
```
